src/main/java/de/uni_tuebingen/ub/nppm/model/SelektionAngabe.java

```diff
@@ -0,0 +1,23 @@
+package de.uni_tuebingen.ub.nppm.model;
+
+import javax.persistence.*;
+import java.util.*;
+import org.hibernate.annotations.CacheConcurrencyStrategy;
+
+@Entity
+@Table(name = "selektion_angabe")
+@Cacheable
+@org.hibernate.annotations.Cache(usage = CacheConcurrencyStrategy.READ_WRITE)
+class SelektionAngabe extends SelektionBezeichnung {
+
+    @ManyToMany(mappedBy = "angaben")
+    private Set<Einzelbeleg> einzelbeleg = new HashSet<>();
```

Comment:
Since this is a set and not a single object, we should rather call it `einzelbelege`. This also means the function should be renamed, `getEinzelbeleg` => `getEinzelbelege`. Moreover i think that we should not provide a setter function for the whole set, instead we should implement only add/remove functions as you did in the Einzelbeleg model class.

Reply:
done